package-lock.json

```diff
@@ -0,0 +1,9692 @@
+{
```

Comment:
Não precisava ter atualizado as dependências. Quando for fazer o ideal seria criar uma PR especificamente para isso.